src/utils/onChainData.ts

```diff
@@ -56,4 +56,24 @@ export async function getLiveGAData(currentEpoch: number , currentEpochEndTime:
   
   return data;
 
+}
+
+export async function getCouncilVote(network: number, proposalId: string, council: any): Promise<any> {
```

Comment:
```suggestion
export async function getCommitteeVotes(network: number, proposalId: string, council: any): Promise<any> {
```

if possible I feel that we should be using the term committee member rather than council, as its more accurate